### DISCUSSION TITLE
Installing SABnzbb after Radarr

### DISCUSSION THREAD

**Discussion by HolgerM1** [community-scripts/ProxmoxVE#183] [Q&A] [answered]
Hi,
I set up a lxc with radarr with the provided script. As a download client is needed to move on, I tried to install SABnzbplus following [https://sabnzbd.org/wiki/installation/install-debian](url) and then this [https://www.linuxbabe.com/debian/install-sabnzbd-usenet-client-debian](url). Unfortunately, all attempts failed with 
![Screenshot 2024-11-11 180117](https://github.com/user-attachments/assets/1f8da964-d17f-4b37-b254-05fadece66d5)
 
Could anybody give me a hint? Even another download client would be fine.

Thanks!

**Comment by MickLesk** [comment, marked as answer]
You need unrar, thats an non free deb package.

```bash
cat <<EOF >/etc/apt/sources.list.d/non-free.list
deb http://deb.debian.org/debian bookworm main contrib non-free non-free-firmware
EOF
apt-get update
apt-get install -y unrar
rm /etc/apt/sources.list.d/non-free.list

**Reply by HolgerM1** [reply]
Thanks!

**Reply by HolgerM1** [reply]
If I may ask one more question?!
I know that's off-topic, but any idea about that?

![Screenshot 2024-11-15 171403](https://github.com/user-attachments/assets/3c2eab4b-7273-4301-8d23-5e5cec01e3f6)
